EarlGrey/Synchronization/GREYUIWebViewIdlingResource.m

```diff
@@ -137,6 +137,14 @@ - (BOOL)isIdleNow {
     }
   }
 
+  // UIWebViews may be used to display PDFs in addition to HTML based content. Test if there's a PDF
+  // view populated in UIWebView's hierarchy.
+  BOOL webViewIsDisplayingPDF = ([[[_webView valueForKey:@"_internal"] valueForKey:@"pdfHandler"]
+                                     valueForKey:@"pdfView"] == nil);
+  if (webViewIsDisplayingPDF) {
```

Comment:
It's not clear to me what's throwing off EarlGrey sync logic.

@tirodkar Can you please see if it's something we can fix in the core-logic rather than short circuiting the entire method when PDFs are being loaded?